site/en/blog/colrv1-fonts/index.md

```diff
@@ -0,0 +1,301 @@
+featuring energetic sweep gradients, made by the innovative type foundry <a
+href="https://underware.nl">Underware</a>, (<a
+href="https://twitter.com/underware">@underware</a> on Twitter). Read more about Underware’s
+first COLRv1 release in their <a
+href="https://www.underware.nl/blog/2022/01/plakato-color/">blog post</a>.
+</figcaption> </figure>
+
+
+### No more image replacement: Emoji Fonts
+
+If you support user generated content, your users probably use emoji. Today it's
+very common to scan text and replace any emoji encountered with images to ensure
+consistent cross-platform rendering and the ability to support newer emojis than
+the OS supports. Those images then have to be switched back to text during
+clipboard operations. Here's a real example:
+
+<figure>
+{% Img src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/dgpLYdWxn0G94lH0rVWI.png", alt="A
+code snippet showing inline images as img tags and metadata as part of a chat history", width="800", height="88" %}
+<figcaption>Image replacement in Google Chat</figcaption>
+</figure>
+
+If you have an emoji font you just render the text in the font, like this:
+
+```html
+<style>
+@import url(https://fonts.googleapis.com/css2?family=Noto+Color+Emoji);
+
+.emoji {
+  font-family: "Noto Color Emoji", sans-serif;
+  font-size: 6em;
+}
+</style>
+<span class="emoji">🙂</span>
+```
+
+Similarly, in an emoji reaction component, COLRv1 offers the opportunity to use
+a compact font file instead of a catalog of image assets.
+
+<figure> {% Img
+src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/dgPBLfMKknICwHOyNgrI.png", alt="Emoji
+picker UI as used on GitHub", width="408", height="390" %} <figcaption>Emoji
+Reaction Picker on GitHub</figcaption> </figure>
+
+Imagine how many images you'd have to fetch for a complete emoji picker!
+
+### Color in Icon Fonts
+
+Using color in icon fonts adds clarity through layers and makes glyphs easier to understand.
+
+<figure>
+{% Img src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/MHHhcD15T0j4q0J3jhPW.png", alt="3 green icons in black outline", width="400", height="354" %}
+<figcaption>
+Material two-tone icons from <a href="https://fonts.google.com/icons">https://fonts.google.com/icons</a>
+</figcaption>
+</figure>
+
+### Artistic Expression
+
+Space-efficient color fonts enable new forms of artistic expression in text on
+the web. This example of a Kufi-style Arabic font uses color gradients as an
+artistic interpretation of what the ink flow of traditional calligraphy could
+look like when applied to the Kufi style of Arabic writing, which originates
+from not being written with nib and ink but carved in stone.
+
+<figure>{% Img
+src="image/roKb5a4ddjOSoIbxRFI9kDeuVvE2/ZfUBAXQMogmAOsLwjcxO.png", alt="Arabic
+letters with gradients from black to red.", width="600", height="218" %}
+<figcaption><a href="https://github.com/aliftype/reem-kufi">Reem Kufi Ink</a>,
+an Arabic font by Khaled Hosny</figcaption> </figure>
+
+## What’s next?
```

Comment:
I'm not sure this heading gets you much. You could probably remove it and promote it's Child sections.

Reply:
Ack, promoted subsections.